usaspending_api/awards/tests/test_awards_v2.py

```diff
@@ -151,92 +195,92 @@ def awards_and_transactions(db):
         "total_subaward_amount": 12345.00,
         "subaward_count": 10,
     }
-    mommy.make('awards.Award', **award_1_model)
-    mommy.make('awards.Award', **award_2_model)
+    mommy.make("awards.Award", **award_1_model)
+    mommy.make("awards.Award", **award_2_model)
 
 
 @pytest.mark.django_db
 def test_award_last_updated_endpoint(client):
     """Test the awards endpoint."""
 
     test_date = datetime.datetime.now()
-    test_date_reformatted = test_date.strftime('%m/%d/%Y')
+    test_date_reformatted = test_date.strftime("%m/%d/%Y")
 
-    mommy.make('awards.Award', update_date=test_date)
-    mommy.make('awards.Award', update_date='')
+    mommy.make("awards.Award", update_date=test_date)
+    mommy.make("awards.Award", update_date="")
 
-    resp = client.get('/api/v2/awards/last_updated/')
+    resp = client.get("/api/v2/awards/last_updated/")
     assert resp.status_code == status.HTTP_200_OK
-    assert resp.data['last_updated'] == test_date_reformatted
+    assert resp.data["last_updated"] == test_date_reformatted
 
 
 @pytest.mark.django_db
 def test_award_endpoint_generated_id(client, awards_and_transactions):
 
-    resp = client.get('/api/v2/awards/ASST_AW_3620_-NONE-_1830212.0481163/')
+    resp = client.get("/api/v2/awards/ASST_AW_3620_-NONE-_1830212.0481163/")
     assert resp.status_code == status.HTTP_200_OK
     assert json.loads(resp.content.decode("utf-8")) == expected_response_asst
 
-    resp = client.get('/api/v2/awards/CONT_AW_9700_9700_03VD_SPM30012D3486/')
+    resp = client.get("/api/v2/awards/CONT_AW_9700_9700_03VD_SPM30012D3486/")
     assert resp.status_code == status.HTTP_200_OK
     assert json.loads(resp.content.decode("utf-8")) == expected_response_cont
 
-    resp = client.get('/api/v2/awards/1/')
+    resp = client.get("/api/v2/awards/1/")
     assert resp.status_code == status.HTTP_200_OK
     assert json.loads(resp.content.decode("utf-8")) == expected_response_asst
 
-    resp = client.get('/api/v2/awards/2/')
+    resp = client.get("/api/v2/awards/2/")
     assert resp.status_code == status.HTTP_200_OK
     assert json.loads(resp.content.decode("utf-8")) == expected_response_cont
 
```

Comment:
You don't test for an IDVResponse with any of these. The award models that you mommy-make have categories of "contract", and "grant", so you never trigger the construct_idv_response functionality.

Reply:
@IsaacRay What about `test_awards_idv_v2.py`?